ui/app/components/ui/circle-icon/circle-icon.component.js

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+export default class CircleIcon extends PureComponent {
+  static propTypes = {
+    size: PropTypes.string,
+    circleClass: PropTypes.string,
+    iconSource: PropTypes.string,
+    iconSize: PropTypes.string,
+  }
+
+  static defaultProps = {
+    size: '56px',
+    iconSize: '18px',
+    circleClass: '',
+    iconSource: null,
+  }
+
+  render () {
+    const {
+      size,
+      circleClass,
+      iconSize,
+      iconSource,
+    } = this.props
+
+    return (
+      <div
+        className="circle-icon__container"
+        style={{
+          height: size,
+          width: size,
+        }}
+      >
+        <div
+          className={`circle-icon__border circle-icon__circle ${circleClass}`}
+          style={{
+            height: size,
+            width: size,
+
+          }}
+        />
+        { iconSource ? (
```

Comment:
I noticed that there is no story for a CircleIcon without an icon. Is this component ever useful in the absence of an icon? 🤔 
Perhaps it would be simpler to remove this branch and make `iconSource` a required prop.

Reply:
Fixed in 3099a26f1